tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1268,3 +1294,137 @@ def _retrieve_clustermgtd_heartbeat_file(remote_command_executor, clustermgtd_co
     return remote_command_executor.run_remote_command(
         f"cat {clustermgtd_conf_path} | grep heartbeat_file_path | cut -d '=' -f2 | xargs"
     ).stdout
+
+
+def _set_insufficient_capacity_timeout(remote_command_executor, insufficient_capacity_timeout, clustermgtd_conf_path):
+    """Disable fast capacity failover by setting insufficient_capacity_timeout < 0."""
+    remote_command_executor.run_remote_command(
+        f"echo 'insufficient_capacity_timeout = {insufficient_capacity_timeout}' | sudo tee -a "
```

Comment:
Added in PR: https://github.com/aws/aws-parallelcluster-cookbook/pull/1420

Reply:
I saw you closed this PR cause we want to keep this enabled by default however I would anyway always set insufficient_capacity_timeout = 600 (the default) in the clustermgtd config so that it’s easier for customer to change and the default is explicit

Reply:
Added